bin/main.rb

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env ruby
+# This is just a representation of how user interface will be, we have been able to capture everything
+# in the next milestone. Please bear with us
+
+class Main
+  def self.welcome
+    puts 'Welcome to the game'
+    puts 'Press 1 to start'
+    choice = gets.chomp
+
+    if choice.to_i == 1
+      choice
+    else
+      'wrong value'
+    end
+  end
+
+  def self.display_board
+    print "\t|\s \s"
+    print "|\s \s"
+    puts "|\sX\s|"
+    puts "\t-------------"
+    print "\t|\sX\s"
+    print "|\sO\s"
+    puts "|\s \s|"
+    puts "\t-------------"
+    print "\t|\sX\s"
+    print "|\s \s"
+    puts "|\sO\s|"
+  end
+
+  def self.enter_position(name)
+    puts "[#{name}] - Enter position (1-9 ):"
+    my_pos = gets.chomp
+    my_pos.to_i
+  end
+
+  def self.statuses(status)
+    puts status
+  end
+end
+
+Main.welcome
+Main.display_board
+
+maximum_moves = 5
+
+loop do
+  Main.enter_position('Player_name')
+  Main.display_board
+  break if maximum_moves == 1
+
+  maximum_moves -= 1
+end
```

Comment:
- use a variable to terminate the loop.
- this should not be complex. you can even use just comments to show you thought process.
- for example,
```ruby
game_on = true
while game_on
# let players take turn until there is a winner
if winner
# tell which player won and end game
end
game_on = false
end
```